packages/grid/docker-compose.yml

```diff
@@ -76,9 +78,48 @@ services:
   #   volumes:
   #     - ./rabbitmq/rabbitmq.conf:/etc/rabbitmq/rabbitmq.conf
 
+  worker:
```

Comment:
SUGGESTION: we could combine the section of worker and backend container, as both of them are equivalent  and it would help us maintain less code.